inc/osvr/Util/StdDeletable.h

```diff
@@ -22,25 +22,27 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#ifndef INCLUDED_StdDeletable_h_GUID_2C03AF21_E4B8_44B3_0F77_630666063711
-#define INCLUDED_StdDeletable_h_GUID_2C03AF21_E4B8_44B3_0F77_630666063711
+#ifndef INCLUDED_Deletable_h_GUID_66D6C45A_7AB7_4198_F8EE_202F0FCE682C
+#define INCLUDED_Deletable_h_GUID_66D6C45A_7AB7_4198_F8EE_202F0FCE682C
```

Comment:
If you rename the file, it's courteous to rename the header guard. In this case, leave it intact.  There's a nifty (well, kind of hacky but useful) boilerplate generator that spits out files like that - think it might be linked to in the CONTRIBUTING file? or maybe HACKING? it's on like tools.getosvr.org and there are some powershell scripts to add it to your right-click send to menu